src/Entities/Entity.cpp

```diff
@@ -332,7 +332,7 @@ void cEntity::TakeDamage(eDamageType a_DamageType, cEntity * a_Attacker, int a_R
 		TDI.Attacker = nullptr;
 	}
 	TDI.RawDamage = a_RawDamage;
-	TDI.FinalDamage = a_FinalDamage;
+	TDI.FinalDamage = static_cast<float>(a_FinalDamage);
```

Comment:
Changing `a_FinalDamage` to a `float` would make sense, too.

Reply:
thx it is simple and good advise